### DISCUSSION TITLE
someone (nicsure) make interface pc to control live the quansheng uv-k5 from egzumer firmware....

### DISCUSSION THREAD

**Discussion by joc2** [egzumer/uv-k5-firmware-custom#390] [Show and tell]
if you want to look at this interface pc....  
it's the starting of the app, but it's maybe be interesting to look at..

https://github.com/nicsure/quansheng-dock-fw
he made a youtube video to show the app...
https://www.youtube.com/watch?v=UwTz5wricmY

**Comment by ErikS-web** [comment]
The firmware is based on EGZUMER. :+1:

And its another project to enjoy.
